.github/workflows/dependabot-ci.yml

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: dependabot-ci
+on:
+  pull_request:
+    types: [opened, ready_for_review]
```

Comment:
I don't think the workflow should be executed on `ready_for_review`.  dependabot does not create PRs as draft.

Reply:
Addressed this in the latest commit.
Removed the `ready_for_review` and added a synchronize parameter as well, since we can also tell dependabot to rebase the PR, so in case such a scenario happens, we would want to trigger the run with the new commit added to the HEAD.